testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -880,14 +842,6 @@
   row.set(true)
 end
 
-When(/^I uncheck "([^"]*)" in the list$/) do |text|
-  top_level_xpath_query = "//div[@class=\"table-responsive\"]/table/tbody/tr[.//td[contains(.,'#{text}')]]//input[@type='checkbox']"
-  row = find(:xpath, top_level_xpath_query, match: :first)
-  raise "xpath: #{top_level_xpath_query} not found" if row.nil?
-
-  row.set(false)
-end
```

Comment:
This step is only used by the step "I uncheck the * client" line 450 which is never called.